plasma_framework/contracts/src/framework/BlockController.sol

```diff
@@ -47,6 +50,7 @@ contract BlockController is Operated, VaultRegistry {
      */
     function submitDepositBlock(bytes32 _blockRoot) public onlyFromVault {
         require(nextDepositBlock < childBlockInterval, "Exceeded limit of deposits per child block interval");
+        require(!_quarantine.isQuarantined(msg.sender), "Vault is quarantined.");
```

Comment:
I would be really conservative on accessing vars from parent contract: it is not easy to tell where this var is from, also when multi inheritance hit in and if ExitGameRegistry has a same named var.....what would happen on the layer of PlasmaFramework (? I guess it should be fine but....who knows XD) 

I'd like to propose just change the modifier `onlyFromVault` to `onlyFromNonQuarantinedVault` and add this line in the modifier.